### PR TITLE
Bugfix/coinstac storage proxy integration

### PR DESCRIPTION
**Pull request by cdaringe**
#### asana
https://app.asana.com/0/search/134376730776913/134376730776925

# problem statement

- pouchdb makes http requests with trailing `/`s. it's a bummer

# solution

make steelpenny great.

# testing

dont bother if you don't want to, but you are welcome to pull the feature and do `npm test`. you might be asking, do tests pass? they do.

screen shot 2016-05-17 at 4 59 30 pm

### CODE REVIEW

File: packages/steelpenny/src/lib/app-routes/coins/scan-details.js [hunk 1]
```diff
@@ -6,51 +6,51 @@ const _ = require('lodash');
 
 const scanController = require('../../controllers/scans.js');
 
-exports.register = function(server, options, next) {
```
Comment (cdaringe) [inline packages/steelpenny/src/lib/app-routes/coins/scan-details.js]: whoops! atom-eslint auto formatted! only one change in this file. will highlight below

File: packages/steelpenny/src/lib/app-routes/coins/scan-details.js [hunk 2]
```diff
@@ -6,51 +6,51 @@ const _ = require('lodash');
...
+exports.register = function (server, options, next) {
 
-    const path = '/scans/{id}/';
-    const Scan = server.plugins.bookshelf.model('Scan');
+  const path = '/scans/{id}';
```
Comment (cdaringe) [inline packages/steelpenny/src/lib/app-routes/coins/scan-details.js]: 👀 lack of trailing `/` here.  this is the sole change to this file. can confirm with `?w=1` if desired

File: packages/steelpenny/src/lib/utils/get-connection-options.js [hunk 3]
```diff
@@ -1,21 +0,0 @@
...
-    return {
-        http: {
-            labels: ['api'],
-            port: parseInt(cliOpts.port, 10) || configPort
-
-        }
-    };
-}
-
-module.exports = getConnectionOptions;
```
Comment (cdaringe) [inline packages/steelpenny/src/lib/utils/get-connection-options.js]: this file has been migrated to the file below

File: packages/steelpenny/src/lib/utils/get-server-settings.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      http: {
+        labels: ['api'],
+        port: parseInt(cliOpts.port, 10) || configPort,
+      },
+    },
+    connections: {
+      router: {
+        stripTrailingSlash: true, // @NOTE pouchdb sends trailing /s :'(
+      },
+    },
```
Comment (cdaringe) [inline packages/steelpenny/src/lib/utils/get-server-settings.js]: ^^ this is the key detail for this PR.

**Comment by swashcap** [comment]
The following tests are failing:

```shell
GET /scans/id/
    1) Should return all scans and series and series data

Users
  POST /users
    2) Posts a new user
    3) Prevents a user with an existing username
    4) Prevents a new user with an existing email
    5) Prevents a new user with an empty password
```

**Comment by swashcap** [comment]
Otherwise, seems fine. :+1:

**Pull request merged**